tests/features/support/steps_clone.js

```diff
@@ -17,173 +18,173 @@ KEY:
+// ? @step(r'I should see that "([^"]+)" is "([^"]+)"')  // What do? Anything with label 
 
 // ? "" is selected/picked?
 // √ the third "blah" checkbox in "bleh" is checked
 
-// √ I arrive at the next page
+// √ I arrive at the next page 
 
 // √ I (don't|can't) continue
 
-// √ the link "([^"]+)" should lead to "([^"]+)"
+// √ the link "([^"]+)" should lead to "([^"]+)" 
 
 // √ the link "([^"]+)" should open in (a new window|the same window)
 
 
 // =========================
 // Navigational
-// =========================
+// ========================= 
 //x @step(r'I exit by taping "([^"]+)"')
 //x @step(r'If I see it, I will tap the link "([^"]+)"')
 
-// ? I tap the ${ specified } link "([^"]+)"
+// ? I tap the ${ specified } link "([^"]+)" 
 
 // Also sometimes not navigational
-// ^ @step(r'I tap the button "([^"]+)"')
+// ^ @step(r'I tap the button "([^"]+)"') 
 // ? I tap "([^"]+)" (can this be a button or link or anything?) (Include 1st, 2nd, etc?)
 // ? @step(r'I tap the "([^"]+)" button')
-// ? @step(r'I tap the link "([^"]+)"')
+// ? @step(r'I tap the link "([^"]+)"') 
 // Maybe go with 'I tap'? Is that possible? Maybe we should stick to the script.
-// √ I tap the (button|link) "([^"]+)"
+// √ I tap the (button|link) "([^"]+)" 
 
-// √ the link "([^"]+)" should open a working page
+// √ the link "([^"]+)" should open a working page 
 
 
 // =========================
 // Interactive
 // =========================
-//x @step(r'I upload the file "([^"]*)"')
+//x @step(r'I upload the file "([^"]*)"') 
 //x @step(r'I reload the screen')
-//x @step(r'I select "([^"]+)" from the menu')
+//x @step(r'I select "([^"]+)" from the menu') 
 //x @step(r'I tap the back button')
 //x @step(r'I tap the question back button')
-//x @step(r'I tap the (first|second|third|fourth|fifth|sixth|seventh|eighth|ninth|tenth) link "([^"]+)"')
+//x @step(r'I tap the (first|second|third|fourth|fifth|sixth|seventh|eighth|ninth|tenth) link "([^"]+)"') 
 //x @step(r'I tap the final link "([^"]+)"')
 
-// ? @step(r'I select "([^"]+)" in the combobox')
+// ? @step(r'I select "([^"]+)" in the combobox') 
 // ~ I pick "([^"]+)" in the { specified } combobox
 
 // ? @step(r'I set the combobox text to "([^"]*)"')
-// ~ I set the {specified} combobox text to "([^"]*)"
+// ~ I set the {specified} combobox text to "([^"]*)" 
 
 //x too vague @step(r'I choose "([^"]+)"')  // dropdown
 // ^ I select "" from the "" dropdown
 // ? I choose "([^"]+)" from the ${ descriptor } dropdown
 // ? I set the "([^"]*)" dropdown to "([^"]*)"
-// ? I pick "" from the "" dropdown?
+// ? I pick "" from the "" dropdown? 
 
 // These could be better differentiated
-// ? @step(r'I select "([^"]+)" as the "([^"]+)"')  // options
+// ? @step(r'I select "([^"]+)" as the "([^"]+)"')  // options 
 // ? @step(r'I select "([^"]+)" as the (first|second|third|fourth|fifth|sixth|seventh|eighth|ninth|tenth) "([^"]+)"')
 // All could be combined with regex?
-// ? I pick ""?
+// ? I pick ""? 
```

Comment:
This would need to account for almost all interaction types and that would probably be too complex and fragile.